<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quickStartDialog</class>
 <widget class="QWidget" name="quickStartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>445</width>
        <height>277</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="vipLabel">
                 <property name="text">
                  <string>My Public-Invite:  </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="whatIsVipBtn">
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/question</normaloff>:/question</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="vipEdit">
               <property name="toolTip">
                <string>Filed with your IP v6 
copy and paste it in &quot;paste invite&quot; field 
 in your's friends computer to connect with him</string>
               </property>
               <property name="statusTip">
                <string>my  ip v6 address</string>
               </property>
               <property name="whatsThis">
                <string>This is IP v6 of working node</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="wwwButton">
               <property name="minimumSize">
                <size>
                 <width>71</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>71</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>www</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/globe-model</normaloff>:/globe-model</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="invateLabel">
                 <property name="text">
                  <string>Paste Invite:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="whatisPasteInv">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/question</normaloff>:/question</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="InviteEdit">
               <property name="toolTip">
                <string>IP v6 and IP v4 of node where you want to connect</string>
               </property>
               <property name="statusTip">
                <string>IP v6 and IP v4 of node where you want to connect</string>
               </property>
               <property name="whatsThis">
                <string>IP v6 and IP v4 of node where
 you want to connect</string>
               </property>
               <property name="maxLength">
                <number>128</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="QrScanButton">
               <property name="minimumSize">
                <size>
                 <width>71</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>71</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Scan QR</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="connectButton">
             <property name="toolTip">
              <string>Connect to node (form &quot;Paset Invate&quot; field)</string>
             </property>
             <property name="text">
              <string>Connect peer!</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/dns</normaloff>:/dns</iconset>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="allowFriendsBox">
               <property name="text">
                <string>Allow-endpoint for friends</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/grooup</normaloff>:/grooup</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="allowStrangersBox">
               <property name="text">
                <string>Allow peering strangers</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/glob-telcom</normaloff>:/glob-telcom</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>My Full-Invite:     </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="whatisFullInviteBtn">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/question</normaloff>:/question</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8"/>
             </item>
             <item>
              <widget class="QLineEdit" name="myInviteEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>IP v6 and IP v4 of this node - place in &quot;Paste invate&quot; field if you want to connect to </string>
               </property>
               <property name="statusTip">
                <string>IP v6 and IP v4 of this node \n necessery to connect to this node</string>
               </property>
               <property name="whatsThis">
                <string>IP v6 and IP v4 of this node 
 necessery to connect to this node</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>28</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="QrZoomButton">
             <property name="minimumSize">
              <size>
               <width>71</width>
               <height>71</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>71</width>
               <height>71</height>
              </size>
             </property>
             <property name="text">
              <string>Qr Code</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
